src/main/scala/thurloe/notification/NotificationMonitor.scala

```diff
@@ -229,6 +229,8 @@ class NotificationMonitorActor(val pollInterval: FiniteDuration,
     s"$fireCloudPortalUrl/#workspaces/${workspaceName.namespace}/${workspaceName.name}"
   def submissionUrl(workspaceName: WorkspaceName, submissionId: String): String =
     s"$fireCloudPortalUrl/#workspaces/${workspaceName.namespace}/${workspaceName.name}/job_history/$submissionId"
+  def billingProjectUrl(billingProjectName: String): String =
+    s"$fireCloudPortalUrl/#billing/$billingProjectName&type=project"
```

Comment:
Is this URL correct? When I look at a billing project in Terra, the URL is `https://app.terra.bio/#billing?selectedName=<billing-project>&type=project`.

Reply:
Fixed in f2c4a28a9f1f55a3ca4a2518c71ec582cc541809